scripts/tests/general/tests.yaml

```diff
@@ -709,3 +709,8 @@
       value: 8
     - name: com.sanctionco.thunder.email.ses.SesEmailService.email-send-success
       value: 2
+
+- name: CHECK HEALTH
+  type: healthcheck
+  disabled: true
```

Comment:
Currently disabled due to an issue with the `aws-ses-local` that we use to test with. Opened a PR to fix it, will enable once that is merged and deployed.